ci-operator/config/openshift/cluster-api/openshift-cluster-api-master.yaml

```diff
@@ -1,19 +1,22 @@
-tag_specification:
-  name: '4.0'
-  namespace: ocp
-promotion:
-  name: "4.3"
-  namespace: ocp
 base_images:
   base:
-    name: '4.0'
+    name: "4.0"
```

Comment:
This looks suspicious, I'll file a follow-up PR to switch this to 4.3

Reply:
Q: Is the image our tool's choice, or developer's?